net.sourceforge.qtpfsgui.LuminanceHDR.yaml

```diff
@@ -0,0 +1,131 @@
+        sha256: cac206e63be68136ef556c2b555df659f45098c159ce24804e9d5e9e0286609e
+
+  - name: openexr
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/openexr/openexr-2.2.1.tar.gz
+        sha256: 8f9a5af6131583404261931d9a5c83de0a425cb4b8b25ddab2b169fbf113aecd
+    cleanup:
+      - /bin
+      - /share/doc
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - "./bootstrap.sh --prefix=/app --with-libraries=atomic,chrono,date_time,filesystem,iostreams,program_options,system,thread"
+      - "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2
+        sha256: 7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7
+
+  - name: libraw
+    buildsystem: autotools
+    config-opts:
+     - --disable-examples
+     - --disable-static
+    sources:
+      - type: archive
+        url: https://www.libraw.org/data/LibRaw-demosaic-pack-GPL2-0.18.8.tar.gz
+        sha256: 0b24bcf7bbb5d13fde58bb071f94dc9354be09bc44b2bba0698493065e99f8da
+        dest: ./LibRaw-demosaic-pack-GPL2
+      - type: archive
+        url: https://www.libraw.org/data/LibRaw-demosaic-pack-GPL3-0.18.8.tar.gz
+        sha256: ffd6916cd66c8101e4e6b589799f256c897748d2fd2486aa34c3705146dbc701
+        dest: ./LibRaw-demosaic-pack-GPL3
+      - type: archive
+        url: https://www.libraw.org/data/LibRaw-0.18.13.tar.gz
+        sha256: cb1f9d0d1fabc8967d501d95c05d2b53d97a2b917345c66553b1abbea06757ca
+    cleanup: [/share/doc]
+
+  - name: gsl
+    sources:
+      - type: archive
+        url: http://ftp.halifax.rwth-aachen.de/gnu/gsl/gsl-2.3.tar.gz
+        sha256: 562500b789cd599b3a4f88547a7a3280538ab2ff4939504c8b4ac4ca25feadfb
+    cleanup:
+      - /bin
+      - /share/info
+      - /share/man
+
+  - name: cfitsio
+    make-args: [shared]
+    sources:
+      - type: archive
+        url: http://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio3450.tar.gz
+        sha256: bf6012dbe668ecb22c399c4b7b2814557ee282c74a7d5dc704eb17c30d9fb92e
+
+  - name: luminance
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/LuminanceHDR/LuminanceHDR/archive/v.2.5.1.tar.gz
+        sha256: 07c02b85a09be1cc4c8940d55a7a2f24c55a071904fedf7c81e621fd0f0ba878
+      - type: file
+        path: net.sourceforge.qtpfsgui.LuminanceHDR.appdata.xml
+      - type: file
+        path: net.sourceforge.qtpfsgui.LuminanceHDR.desktop
+      - type: file
+        path: net.sourceforge.qtpfsgui.LuminanceHDR.svg
+    post-install:
+      - install -Dm 644 net.sourceforge.qtpfsgui.LuminanceHDR.appdata.xml -t /app/share/appdata
+      - install -Dm 644 net.sourceforge.qtpfsgui.LuminanceHDR.desktop -t /app/share/applications
+      - install -Dm 644 net.sourceforge.qtpfsgui.LuminanceHDR.svg -t /app/share/icons/hicolor/scalable/apps
```

Comment:
You might want to refresh the icon cache here, otherwise the icon might not show up